frontend/src/libs/components/components.ts

```diff
@@ -1,6 +1,7 @@
 export { App } from './app/app.js';
 export { AudioPlayer } from './audio-player/audio-player.js';
-export { BackButton } from './backward-button/backward-button.js';
+export { BackButtonWrapper } from './backward-button/back-button-wrapper.js';
+export { BackButton } from './backward-button/components/backward-button.js';
```

Comment:
Name of the file and component should be the same
Don't forget about folder name

Reply:
Thanks for your attention!